checkout/interfaces/controller/checkoutcontroller.go

```diff
@@ -298,6 +298,7 @@ func (cc *CheckoutController) SuccessAction(ctx context.Context, r *web.Request)
 				PlacedOrderInfos:    placeOrderFlashData.PlacedOrderInfos,
 			}
 
+			cc.orderService.ClearLastPlacedOrder(ctx)
```

Comment:
we should clear the last placed order in the else part of the placeOrderAction

Reply:
changed as discussed